openmetadata-ui/src/main/resources/ui/playwright/utils/user.ts

```diff
@@ -168,3 +186,520 @@ export const hardDeleteUserProfilePage = async (
 
   await toastNotification(page, /deleted successfully!/);
 };
+
+export const editDisplayName = async (page: Page, editedUserName: string) => {
+  await page.click('[data-testid="edit-displayName"]');
+  await page.fill('[data-testid="displayName"]', '');
+  await page.type('[data-testid="displayName"]', editedUserName);
+
+  const saveResponse = page.waitForResponse('/api/v1/users/*');
+  await page.click('[data-testid="inline-save-btn"]');
+  await saveResponse;
+
+  // Verify the updated display name
+  const userName = await page.textContent('[data-testid="user-name"]');
+
+  expect(userName).toContain(editedUserName);
+};
+
+export const editTeams = async (page: Page, teamName: string) => {
+  await page.click('[data-testid="edit-teams-button"]');
+  await page.click('.ant-select-selection-item-remove > .anticon');
+
+  await page.click('[data-testid="team-select"]');
+  await page.type('[data-testid="team-select"]', teamName);
+
+  // Click the team from the dropdown
+  await page.click('.filter-node > .ant-select-tree-node-content-wrapper');
+
+  const updateTeamResponse = page.waitForResponse('/api/v1/users/*');
+  await page.click('[data-testid="inline-save-btn"]');
+  await updateTeamResponse;
+
+  // Verify the new team link is visible
+  await expect(page.locator(`[data-testid="${teamName}-link"]`)).toBeVisible();
+};
+
+export const editDescription = async (
+  page: Page,
+  updatedDescription: string
+) => {
+  await page.click('[data-testid="edit-description"]');
+
+  // Clear and type the new description
+  await page.locator(descriptionBox).fill(updatedDescription);
+
+  const updateDescription = page.waitForResponse('/api/v1/users/*');
+  await page.click('[data-testid="save"]');
+  await updateDescription;
+
+  await page.click('.ant-collapse-expand-icon > .anticon > svg');
+
+  // Verify the updated description
+  const description = page.locator(
+    ':nth-child(2) > :nth-child(1) > [data-testid="viewer-container"] > [data-testid="markdown-parser"] > :nth-child(1) > .toastui-editor-contents > p'
```

Comment:
i think, we need to re-think here for selectors! we have description const can you check that?